bundles/org.openhab.binding.vizio/src/main/resources/OH-INF/thing/vizio.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vizio"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Vizio TV Thing -->
+	<thing-type id="vizio_tv">
+		<label>Vizio TV</label>
+		<description>
+			A Vizio SmartCast TV
+		</description>
+
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="source" typeId="source"/>
+			<channel id="activeApp" typeId="activeApp"/>
+			<channel id="control" typeId="control"/>
+			<channel id="button" typeId="buttonTv"/>
+		</channels>
+
+		<properties>
+			<property name="Model Number">unknown</property>
+		</properties>
+
+		<representation-property>uuid</representation-property>
+
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<context>network-address</context>
+				<label>Host Name/IP Address</label>
+				<description>Host Name or IP Address of the Vizio TV</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="true">
+				<label>Port</label>
+				<description>Port for the Vizio TV</description>
+				<default>7345</default>
+				<limitToOptions>true</limitToOptions>
+				<options>
+					<option value="7345">7345 (Newer Models)</option>
+					<option value="9000">9000 (Older Models)</option>
+				</options>
+			</parameter>
+			<parameter name="authToken" type="text" required="false">
+				<label>Auth Token</label>
+				<description>Auth Token that is obtained via the pairing process; See README for details</description>
+			</parameter>
+			<parameter name="appListJson" type="text" required="false">
+				<context>script</context>
```

Comment:
This does not seem to be a supported context - see https://www.openhab.org/docs/developer/bindings/config-xml.html#supported-contexts

Reply:
I forgot where I first saw this, but it currently exists in the _dbquery_ binding. And I added it to _nuvo_ also where it is was handy for editing large XML strings in thing configurations.
![image](https://user-images.githubusercontent.com/26532542/205757944-24d2f2f5-2544-4d3d-be21-ad668c06375a.png)

![image](https://user-images.githubusercontent.com/26532542/205758216-d8e3c2ef-9e0b-422b-b8f8-10ac2fc08a4d.png)

Reply:
You are right, dbquery and nuvo use it also, so maybe it's simply missing in the documentation. I'll check this tomorrow.